code/datums/station_traits/positive_traits.dm

```diff
@@ -359,5 +361,14 @@
 	show_in_report = TRUE
 	blacklist = list(/datum/station_trait/colored_assistants)
 
-#undef PARTY_COOLDOWN_LENGTH_MIN
-#undef PARTY_COOLDOWN_LENGTH_MAX
+/// Crew don't ever spawn as enemies of the station. Obsesseds, blob infection, space changelings etc can still happen though
+/datum/station_trait/background_checks
+	name = "Station-Wide Background Checks"
+	report_message = "We're testing a new protocol for station-wide background checks, take comfort in trusting your fellow crew."
```

Comment:
'testing' is an unfortunate choice of words because tests imply uncertainty and failure chance

how about

```suggestion
	report_message = "We replaced the intern doing your crew's background checks with a trained screener for this shift! That said, our enemies may just find another way to infiltrate the station, so be careful."
```